src/main/java/duke/commands/FindCommand.java

```diff
@@ -0,0 +1,44 @@
+package src.main.java.duke.commands;
+
+import java.util.List;
+
+import src.main.java.duke.commons.Messages;
+import src.main.java.duke.data.task.Task;
+
+/**
+ * Represents an find command. Upon execution, list the tasks that contain the word to user.
+ */
+public class FindCommand extends Command {
+
+    public static final String COMMAND_WORD = "find";
+
+    private static final String INDENT = "   ";
+    // Message to add
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Find the tasks that contains the words. \n"
+            + "Parameters: find WORD\n" + "Example: " + COMMAND_WORD + " 1";
+
```

Comment:
Your code is quite clean in general, but do note that javadocs for all public classes and methods is required.